tests/unit/test_tokenization_methods.py

```diff
@@ -163,3 +163,113 @@ def test_get_token_position_int_1_pos():
+          1507, 29229,   276, 29229,    65,  3434,    28,   477,  1800,   826,
+           311, 29229,  2063,   282,  1056, 10391,  1296,   477,   603,  1052,
+           494,    10, 40501,  2192,   647,  1800,   826,   311, 29229,  2063,
+           282,  1056, 10391,  1296,   647, 29229,    10,    19,    18,    14,
+           607,    11,   210,  1800,  7189,   436,  3175,   311,  3469,   324,
+           512, 29229,  3357,   188,   188,     5,   826,   311, 29229,  2063,
+           282,  1056, 10391,  1296],
+        [    1, 24684,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2,     2,     2,     2,     2,     2,     2,
+             2,     2,     2,     2]]).tolist()
+    # fmt: on
```

Comment:
I'd suggest using https://pypi.org/project/snapshottest/ to avoid having a huge tensor in the test file (the snapshot can be added manually to make sure it's correct v. 1.6.1). 

Also in this case you could just test the entire vocab space (as file length is no longer a problem)?

Reply:
Done. However 

1. I don't know how to setup manually snapshotting 1.6.1
2. I had to `pip install snapshottest`. Should this library be in our `pyproject.toml`?

Reply:
Yep `poetry add --group dev snapshottest`

If you run it once it should create a snapshot directory in the tests directory with the snapshot in it. It'll just put the tensor in there so you can overwrite it if you want to manually change it

Reply:
Oh @alan-cooney you want me to commit the file with the tensor?